### ISSUE TITLE
Checkbox for Voicebox support [EDIT: and fix detection via IN 65407 BAND 254]

### ISSUE THREAD

**Issue by dandoore** [simonowen/simcoupe#40]
Can the Voicebox support be made optional/disable-able? - I was trying to test the routine to see if was present and couldn’t :)

**Comment by stefandrissen** [comment]
While the user interface option is still missing, I expect that you can configure this in ``%appdata%\simcoupe.cfg`` 
Change line:

``voicebox=1``

to:

``voicebox=0``

**Comment by dandoore** [comment]
Interesting, with it disabled in the options it passes the detection test (which should fail):

```
10 LET inport=IN 65407,sig=inport BAND 254
20 IF sig<>24 THEN CLS : PRINT AT 10,6;"No Voice Box Present" 
```

But then stays in 'busy' state:

```
30 LET busy=IN 65407 BAND 1
40 IF busy=1 THEN GO TO 40
50 PRINT "Ready"
```

So the routines hang there.  This is kind of mired by the fact I can't test on real hardware but it should fail the detection routine.

**Comment by dandoore** [comment]
Attached disk image:
[Voicebox-SPT-Program.zip](https://github.com/simonowen/simcoupe/files/5990557/Voicebox-SPT-Program.zip)

**Comment by simonowen** [comment]
I've pushed a quick fix for the detection part of this to the `dev` branch. I'll look at creating an updated build once I've been through some of the other outstanding issues.

The voicebox option only covered whether its sample output was mixed into the final sound output, so the underlying device still responded to I/O as normal. I've suppressed the port reading when the voicebox option is disabled, but still allow the writes to go through. This should ensure the device is in a good state should it later be re-enabled during emulator use.

I'll leave the issue open for the original checkbox request as it's probably not so quick to add. I haven't reinstalled Visual Studio on my new PC but last time I tried to edit the Windows .rc file I got an error that Microsoft said they weren't going to fix. The built-in interface should probably have the equivalent checkbox change but I want to scrap all of that -- a whole other can of refactoring worms. For now you can change the `%APPDATA%\SimCoupe\SimCoupe.cfg` file as Stefan suggested, which requires a relaunch to pick up.

I've been snowed under with work and other things, and home development has been the last thing I've felt like in my spare time recently. I've now got enough slack to work through the backlog and should be able to fix any minor issues once my development setup is restored. I'll probably not be tackling any big code changes until things are quieter and I get my home development mojo back. Thanks @stefandrissen for holding the fort here!

**Comment by simonowen** [comment]
Fixed in 1470158519f05e28c0b800c9d73fe66e76403b6f is part of the [20210501 build](https://github.com/simonowen/simcoupe/releases/tag/20210501).
